app/javascript/components/MainContainer/Filters/Filters.js

```diff
@@ -12,43 +16,87 @@ class Filters extends Component {
             cuisine_filter_selected: 'all',
             ten_bis_filter_selected: 'all'
         }
+    }
+
+    onChangeRating = (item) => {
+        this.setState({rating_filter_selected: item.value})
+        this.props.onChangeFilter('rating_filter', item.value);
```

Comment:
What happens if you forget to set the `onChangeFilter` prop and it's not a function?

Reply:
everything fails :)